src/main/java/com/siit/hospital_manager/controller/AppointmentController.java

```diff
@@ -0,0 +1,146 @@
+package com.siit.hospital_manager.controller;
+
+import com.siit.hospital_manager.model.Appointment;
+import com.siit.hospital_manager.model.AppointmentStatus;
+import com.siit.hospital_manager.model.dto.AppointmentDto;
+import com.siit.hospital_manager.model.dto.CreateAppointmentDto;
+import com.siit.hospital_manager.model.dto.UpdateAppointmentDto;
+import com.siit.hospital_manager.service.AppointmentService;
+import com.siit.hospital_manager.service.EmailSenderService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.security.Principal;
+import java.util.List;
+
+import static com.siit.hospital_manager.util.AuthUtils.*;
+import static com.siit.hospital_manager.util.AuthUtils.isDoctor;
+
+
+@Controller
+@RequestMapping("/appointment")
+
+@RequiredArgsConstructor
+public class AppointmentController {
+
+    private final AppointmentService appointmentService;
+    private final EmailSenderService emailSenderService;
+
+    @GetMapping("/findAllByUserName")
+    public String findAllByUserName(Model model, Principal principal, Authentication authentication, AppointmentStatus appointmentStatus) {
```

Comment:
appointmentStatus doesn't seem to be used here, and the same for the next two methods